android/app/src/main/java/im/status/ethereum/MainApplication.java

```diff
@@ -28,6 +28,8 @@
 import com.facebook.react.bridge.JSIModulePackage;
 import com.swmansion.reanimated.ReanimatedJSIModulePackage;
 
+import org.devio.rn.splashscreen.SplashScreenReactPackage;
```

Comment:
could you pls elaborate on why only import is needed ?

Reply:
Oops, this is redundant with react native auto linking. I'll remove it.